### ISSUE TITLE
Error: infinite rendering invalidation detected

### ISSUE THREAD

**Issue by Turbo87** [skylines-project/skylines#540]
https://sentry.io/skylines/skylines/issues/176050315/

```
Error: infinite rendering invalidation detected
  at callbacks (bower_components/ember/ember.prod.js:10511)
  at this (bower_components/ember/ember.prod.js:625)
  at this (bower_components/ember/ember.prod.js:601)
  at this (bower_components/ember/ember.prod.js:704)
  at backburner (bower_components/ember/ember.prod.js:735)
...
(45 additional frame(s) were not displayed)
```

**Comment by Turbo87** [comment]
see https://github.com/emberjs/ember.js/issues/14351

This was apparently fixed already and release in Ember `v2.10.0-beta.2`, but we are still using `beta.1`.

**Comment by Turbo87** [comment]
Reason for running the `beta` version is that it includes the [Glimmer 2](https://github.com/tildeio/glimmer) rendering engine, which is faster and reduces the template sizes significantly (`skylines.js` went from 1.3MB to 700KB).
